api/src/oscc.c

```diff
@@ -496,7 +496,7 @@ oscc_result_t oscc_can_write( long id, void *msg, unsigned int dlc )
         tx_frame.can_dlc = dlc;
         memcpy( tx_frame.data, msg, dlc);
 
-        int result = write( can_socket, &tx_frame, sizeof(tx_frame ) );
+        int result = write( can_socket, &tx_frame, sizeof(tx_frame ));
```

Comment:
Was this format change intentional? (also line 497 looks like it's missing a space)

Reply:
Nope, that was from a problematic merge.

d73e3b2